src/Domain/CoordinateMaps/UniformCylindricalSide.hpp

```diff
@@ -0,0 +1,606 @@
+ * \frac{\partial\sin\phi}{\partial\bar{x}}
+ *  &= -\frac{\bar{x}\bar{y}}{\bar{R}^3\bar{\rho}^3},
+ *  \label{eq:dsinphidxbar} \\
+ * \frac{\partial\sin\phi}{\partial\bar{y}}
+ *  &= \frac{\bar{x}^2}{\bar{R}^3\bar{\rho}^3},
+ *  \label{eq:dsinphidybar}
+ * \f}
+ * and we know that \f$\phi\f$ is independent of \f$\bar{z}\f$.
+ *
+ * Finally, from Eqs. (\f$\ref{eq:defrhobar}\f$) and
+ * (\f$\ref{eq:lambdafromrhobar}\f$) we have
+ *
+ * \f{align}
+ * \frac{\partial\lambda}{\partial\bar{x}}
+ *  &= \frac{\bar{x}}{\bar{R}^2\bar{\rho}},
+ *  \label{eq:dlambdadxbar} \\
+ * \frac{\partial\lambda}{\partial\bar{y}}
+ *  &= \frac{\bar{y}}{\bar{R}^2\bar{\rho}},
+ *  \label{eq:dlambdadybar}
+ * \f}
+ * with no dependence on \f$\bar{z}\f$.
+ *
+ * Putting these results together yields
+ * \f{align}
+ * \frac{\partial x^0}{\partial \bar{x}} &=
+ * \frac{\bar{y}^2}{\bar{\rho}^3\bar{R}^3}R_1\sin\theta_1 +
+ *        (R_2\sin\theta_2-R_1\sin\theta_1)
+ *        \frac{\lambda \bar{R}^2\bar\rho^2+\bar{x}^2}{\bar\rho^3\bar{R}^3}
+ *  + \frac{\bar{x}}{\bar\rho\bar{R}^2}(C_2^0-C_1^0),\\
+ * \frac{\partial x^0}{\partial \bar{y}} &=
+ * \frac{\bar{x}\bar{y}}{\bar{\rho}^3\bar{R}^3}
+ *        (R_2\sin\theta_2-2 R_1\sin\theta_1)
+ *  + \frac{\bar{y}}{\bar\rho\bar{R}^2}(C_2^0-C_1^0),\\
+ * \frac{\partial x^0}{\partial \bar{z}} &=
+ * -\frac{1}{2}\frac{\bar{x}}{\bar\rho\bar{R}}\left[
+ *   \cot\theta_1(1-\lambda)(z^{+}_{\mathrm{P}1}-z^{-}_{\mathrm{P}1})+
+ *   \cot\theta_2\lambda(z^{+}_{\mathrm{P}2}-z^{-}_{\mathrm{P}2})\right],\\
+ * \frac{\partial x^1}{\partial \bar{x}} &=
+ * \frac{\bar{x}\bar{y}}{\bar{\rho}^3\bar{R}^3}
+ *        (R_2\sin\theta_2-2 R_1\sin\theta_1)
+ *  + \frac{\bar{x}}{\bar\rho\bar{R}^2}(C_2^1-C_1^1),\\
+ * \frac{\partial x^1}{\partial \bar{y}} &=
+ * \frac{\bar{x}^2}{\bar{\rho}^3\bar{R}^3}R_1\sin\theta_1 +
+ *        (R_2\sin\theta_2-R_1\sin\theta_1)
+ *        \frac{\lambda \bar{R}^2\bar\rho^2+\bar{y}^2}{\bar\rho^3\bar{R}^3}
+ *  + \frac{\bar{y}}{\bar\rho\bar{R}^2}(C_2^1-C_1^1),\\
+ * \frac{\partial x^1}{\partial \bar{z}} &=
+ * -\frac{1}{2}\frac{\bar{y}}{\bar\rho\bar{R}}\left[
+ *   \cot\theta_1(1-\lambda)(z^{+}_{\mathrm{P}1}-z^{-}_{\mathrm{P}1})+
+ *   \cot\theta_2\lambda(z^{+}_{\mathrm{P}2}-z^{-}_{\mathrm{P}2})\right],\\
+ * \frac{\partial x^2}{\partial \bar{x}} &=
+ *  \frac{\bar{x}}{\bar\rho\bar{R}^2}\left(
+ *        C_2^2-C_1^2 + R_2\cos\theta_2-R_1\cos\theta_1\right),\\
+ * \frac{\partial x^2}{\partial \bar{y}} &=
+ *  \frac{\bar{y}}{\bar\rho\bar{R}^2}\left(
+ *        C_2^2-C_1^2 + R_2\cos\theta_2-R_1\cos\theta_1\right),\\
+ * \frac{\partial x^2}{\partial \bar{z}} &=
+ * \frac{1}{2}(1-\lambda)(z^{+}_{\mathrm{P}1}-z^{-}_{\mathrm{P}1})+
+ * \frac{1}{2}\lambda(z^{+}_{\mathrm{P}2}-z^{-}_{\mathrm{P}2}).
+ * \f}
+ *
+ * ### inv_jacobian
+ *
+ * The inverse Jacobian is computed by numerically inverting the
+ * Jacobian.
+ *
+ * ### Restrictions on map parameters
+ *
+ * We demand that Sphere 1 is fully contained inside Sphere 2, and
```

Comment:
Ahh ok here's the condition on which sphere contains which. Still might be good to also mention this at the beginning.